server/base.py

```diff
@@ -0,0 +1,113 @@
+from bson.objectid import ObjectId
+import jsonschema
+
+from girder import logger as log
+from girder.constants import AssetstoreType, SettingKey
+from girder.exceptions import ValidationException
+from girder.models.assetstore import Assetstore
+from girder.utility import setting_utilities
+
+
+DB_INFO_KEY = 'databaseMetadata'
+
+# This is a quasi-random ObjectID with an 'old' date
+DB_ASSETSTORE_ID = '4b3d3b0193371b154c68f931'
```

Comment:
I know I've said this in the past, but I'll reiterate that I think the capabilities of this plugin would be better exposed through a Database or Table model rather than as an Assetstore. Of course, this isn't the time to refactor / rewrite the plugin, so no action is required here, but my concern about the fit within the assetstore abstraction still stands.